tests/ed25519.rs

```diff
@@ -112,11 +109,19 @@ mod vectors {
 
         let sig2: Signature = keypair.sign_prehashed(prehash_for_signing, None).unwrap();
 
-        assert!(sig1 == sig2,
-                "Original signature from test vectors doesn't equal signature produced:\
-                \noriginal:\n{:?}\nproduced:\n{:?}", sig1, sig2);
-        assert!(keypair.verify_prehashed(prehash_for_verifying, None, &sig2).is_ok(),
-                "Could not verify ed25519ph signature!");
+        assert!(
+            sig1 == sig2,
+            "Original signature from test vectors doesn't equal signature produced:\
+                \noriginal:\n{:?}\nproduced:\n{:?}",
```

Comment:
Weirdly indented. Is this a norm I don't know about?

Reply:
Nah, just the result of rustfmt being run across the code